server/mscalendar/settings_notifications.go

```diff
@@ -35,6 +35,7 @@ func (s *notificationSetting) Set(userID string, value interface{}) error {
 	cal := s.getCal(userID)
 
 	if boolValue {
+		// REVIEW: notification subscription logic in notification settings. this seems a bit weird. what does this function/block have to do specifically with notifications?
 		_, err := cal.LoadMyEventSubscription()
 		if err != nil {
```

Comment:
Not really sure what's going on here. It seems like:

"If a setting is a boolean, create an event subscription if not exists"

This is `notificationSetting.Set`, so I suppose this is "I want to change notification settings". what do we do when the value hasn't changed? I suppose this function takes care of no-ops